### PR TITLE
Team3 aya new

### PR DESCRIPTION
**Pull request by Aya-Alabrash**

### CODE REVIEW

File: pages/apply/upload/index.vue [hunk 1]
```diff
@@ -222,16 +223,33 @@ export default {
...
+      }
     },
+
+    isValidEmail() {
+      var re = /^([a-zA-Z0-9_\.\-])+\@(([a-zA-Z0-9\-])+\.)+([a-zA-Z0-9]{2,4})+$/;
+      if (!re.test(email.value)) {
+        email.parentNode.classList.remove("active");
+        email.parentNode.classList.add("active");
+        email.value = "Invalid Email";
+        email.focus;
```
Comment (M3kH) [inline pages/apply/upload/index.vue]: What does this line?

File: pages/apply/upload/index.vue [hunk 2]
```diff
@@ -222,16 +223,33 @@ export default {
...
+      // console.log(this.isValidEmail(email.value));
+      if (this.isValidEmail() === true) {
+        this.emailData = email;
+        email.value = this.emailData.value;
+      }
     },
+
+    isValidEmail() {
+      var re = /^([a-zA-Z0-9_\.\-])+\@(([a-zA-Z0-9\-])+\.)+([a-zA-Z0-9]{2,4})+$/;
+      if (!re.test(email.value)) {
```
Comment (M3kH) [inline pages/apply/upload/index.vue]: Email is undefined here. Where does come from?

File: pages/apply/upload/index.vue [hunk 3]
```diff
@@ -222,16 +223,33 @@ export default {
       }
     },
     handleEmail() {
-      this.emailData = email;
-      email.value = this.emailData.value;
+      // console.log(this.isValidEmail(email.value));
+      if (this.isValidEmail() === true) {
+        this.emailData = email;
+        email.value = this.emailData.value;
```
Comment (M3kH) [inline pages/apply/upload/index.vue]: what is email? and what is this method suppose to do?

File: pages/apply/upload1/index.vue [hunk 4]
```diff
@@ -139,12 +141,27 @@ export default {
     },
 
     // Handles a change on the url upload
-    handleUrlUpload() {
-      this.urlData = url;
-      url.value = this.urlData.value;
+    handleFileUpload() {
+      if (this.checkUrl() === true) {
+        this.urlData = url;
+        url.value = this.urlData.value;
```
Comment (M3kH) [inline pages/apply/upload1/index.vue]: This looks like the method email above, where is `url` declared? what this method is suppose to do?

File: pages/apply/upload1/index.vue [hunk 5]
```diff
@@ -139,12 +141,27 @@ export default {
...
+      }
+    },
+    checkUrl() {
+      var inputUrl = url.value;
+      var pattern = /(http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?/;
+      if (!pattern.test(inputUrl)) {
+        url.parentNode.classList.remove("active");
+        url.parentNode.classList.add("active");
+        url.value = "Invalid URL";
+        url.focus;
```
Comment (M3kH) [inline pages/apply/upload1/index.vue]: A lot of copy and pasta 🍝, I could do the same with my comments :-)
So this is same as mention above :-)

File: pages/apply/upload/index.vue (hunk 1 above)
Comment (gisaac85) [reply pages/apply/upload/index.vue]: i deleted line 236+237 ... 
line 238: showing "Invalid Email" in the email input

File: pages/apply/upload/index.vue (hunk 2 above)
Comment (gisaac85) [reply pages/apply/upload/index.vue]: @M3kH What do you mean ?
We used this Regx to make inputed Email correct.

File: pages/apply/upload/index.vue (hunk 3 above)
Comment (gisaac85) [reply pages/apply/upload/index.vue]: @M3kH  checking first if the inputed email is valid then handle it. 
email is : this.$refs.email

File: pages/apply/upload1/index.vue (hunk 4 above)
Comment (gisaac85) [reply pages/apply/upload1/index.vue]: the same above!

File: pages/apply/upload1/index.vue (hunk 5 above)
Comment (gisaac85) [reply pages/apply/upload1/index.vue]: same above!

**Pull request closed without merging**